wallet/storage/src/lib.rs

```diff
@@ -60,6 +63,7 @@ pub trait WalletStorageReadLocked {
         &self,
         account_id: &AccountId,
     ) -> Result<BTreeMap<AccountWalletTxId, WalletTx>>;
+    fn get_signed_transactions(&self) -> Result<Vec<SignedTransaction>>;
```

Comment:
I would call them `user_transactions`, and would add a definition (docs) defining what that means: "A transaction created by this wallet".

Reply:
changed